README.md

```diff
@@ -1 +1,86 @@
-# jenkins-kubernetes-pipeline
\ No newline at end of file
+# Prerequisites 
+Before you begin, make sure you have the following:
+
+* 1- Installed Jenkins on the local system or any VM. [how to install](https://www.jenkins.io/doc/book/installing/)
+     [for add credentials](https://www.jenkins.io/doc/book/using/using-credentials/#:~:text=From%20the%20Jenkins%20home%20page,Add%20Credentials%20on%20the%20left.) [for jenkins plugins](https://www.jenkins.io/doc/book/managing/plugins/)
+* 2- Installed Kubernetes minikube. [how to install](https://kubernetes.io/docs/setup/)
+* 3- Installed Docker.[how to install](https://docs.docker.com/engine/install/)
+* 4- Make sure Jenkins has a connection with Kubernetes  [how to make connection](https://medium.com/@devayanthakur/minikube-configure-jenkins-kubernetes-plugin-25eb804d0dec)
+* 5- Dockerhub repo
+
+# Detail about files that are placed on the repository for Containerize.
+
+* this repo we have `Dockerfile` and `index.php` file. these files are for creating php web page.
```

Comment:
Add this change at end after pipeline trigger this page should be show in browser